src/portal/src/app/base/left-side-nav/system-robot-accounts/system-robot-util.ts

```diff
@@ -92,6 +92,11 @@ export const INITIAL_ACCESSES: FrontAccess[] = [
         "resource": "scan",
         "action": "create",
         "checked": true
+    },
+    {
+        "resource": "scan",
+        "action": "delete",
```

Comment:
stop scan action has changed from “delete" to "stop" (a new action type being added), you may want to update it as well. 
please refer to "Robot account permission" the proposal for more details: 
`https://github.com/goharbor/community/blob/4cb9f33e578cc2cf59c6b89d0172efebefb33746/proposals/new/stop-scan-job-proposal.md#robot-account-permission`
`https://github.com/goharbor/community/blob/4cb9f33e578cc2cf59c6b89d0172efebefb33746/proposals/new/stop-scan-job-proposal.md#robot-account-permission-1`

Reply:
Updated